src/popup/router/components/Modals/TipUrlStatus.vue

```diff
@@ -1,11 +1,13 @@
 <template>
```

Comment:
The whole file should be simplified using Default component. Also we will not doube the future changes made to the Default component here.